contracts/asset-proxy/src/artifacts.ts

```diff
@@ -0,0 +1,29 @@
+// This file is auto-generated by contracts-gen. Don't edit manually.
```

Comment:
```suggestion
/*
 * -----------------------------------------------------------------------------
 * Warning: This file is auto-generated by `contracts-gen`. Don't edit manually.
 * -----------------------------------------------------------------------------
 */
```

Can we do this so that it's physically bigger and less likely someone doesn't see it?